### PR TITLE
add user link in show team command

### PR DESCRIPTION
**Pull request by harryi3t**
Fixes: https://github.com/harryi3t/slackAssist/issues/48

### CODE REVIEW

**Comment by harryi3t** [comment]
test comment

**Comment by harryi3t** [comment]
tes comaand

File: commands/showTeam.js [hunk 1]
```diff
@@ -24,7 +24,9 @@ function _getTeam(callback){
     else{
       var users = '';
       team.forEach(function(user){
-        users += '\n*' + user.name + '* `' + user.username + '`';
+        // *Name* <https://github.com/username|username>
+        users += '\n*' + user.name + '* <' + 
+        "https://github.com/" + user.username + '|' + user.username + '>';
       });
```
Comment (harryi3t) [inline commands/showTeam.js]: test comment

**Comment by balavignesh-s** [comment]
test comment

File: commands/showTeam.js [hunk 2]
```diff
@@ -24,7 +24,9 @@ function _getTeam(callback){
     else{
```
Comment (balavignesh-s) [inline commands/showTeam.js]: something else

Comment (harryi3t) [inline commands/showTeam.js]: something else

**Comment by harryi3t** [comment]
test comment on pull page in converssation

**Comment by harryi3t** [comment]
test comment on pull page in converssation

**Comment by harryi3t** [comment]
test comment on pull page in converssation

**Comment by harryi3t** [comment]
@harryi3t test comment

**Comment by harryi3t** [comment]
test

**Pull request merged**